### PR TITLE
162633493 Add PostgreSQL DB engine and deploy to Heroku

### PR DESCRIPTION
**Pull request by mubstimor**
### What does this PR do?
- Configures Authors Haven to use PostgreSQL as DB Engine
- Sets up and deploys base application to Heroku

#### Task to be completed
- Add Django postresql dependencies to repository
- Add white noise to manage storage of static files in production.

#### How should this be manually tested?
- To run locally set a local environment variable with EXPORT for DATABASE_URL=postgres://user:password@localhost:5432/database in your python environment.
- Visit [here](https://ah-backend-valkyrie.herokuapp.com) for Production version
- Visit [here](https://ah-backend-valkyrie-staging.herokuapp.com) for staging version

#### What are the relevant pivotal tracker stories?
[162633493](https://www.pivotaltracker.com/story/show/162633493)
[162633492](https://www.pivotaltracker.com/story/show/162633492)

### CODE REVIEW

**Comment by araaliFarooq** [comment]
LGTM

**Comment by jeanjoe** [comment]
LGTM

**Comment by mycok** [comment]
LGTM

**Comment by NabunyaLilian** [comment]
LGTM

**Comment by fatukunda** [comment]
LGTM

**Comment by Qadriyah** [comment]
LGTM

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[report]
+omit = *authors/settings.py*
```
Comment (lym) [inline .coveragerc]: Add a new line here.

File: Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+web: python manage.py makemigrations; python manage.py migrate; gunicorn authors.wsgi
```
Comment (lym) [inline Procfile]: Add a new line here

**Comment by lym** [comment]
@mubstimor Please address all the comments and also squash all your commits into one.

**Pull request merged**